functions/system.bash

```diff
@@ -554,3 +554,31 @@ prepare_serial_port() {
 
   whiptail --title "Operation successful" --msgbox "$successText" 9 80
 }
+
+
+## Function for installing the ELElabs EZSP firmware update utility
+## Useful to flash various ZigBee controllers that have an Ember chipset.
+##
+##    ezspUtility_setup
+##
+##
+ezspUtility_setup() {
+  local repo=https://github.com/Elelabs/elelabs-zigbee-ezsp-utility
+  local destdir=/usr/share/openhab
+  local target=Elelabs_EzspFwUtility.py
```

Comment:
Remember to double quote variables, we established that as a coding guideline a while back.

Reply:
ok